RealmSwift/Sync.swift

```diff
@@ -268,7 +268,7 @@ import Combine
 /// This handler is executed on a non-main global `DispatchQueue`.
 @dynamicMemberLookup
 @frozen public struct Functions {
-    weak var user: User?
+    var user: User?
```

Comment:
This should now be `private let user: User`. No reason for the user to be nullable if it's not weak.

Reply:
Done.